packages/calcite-components/src/components/tab/tab.scss

```
@@ -1,3 +1,11 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tab-content-space: Specifies the padding of the block `default` slot.
```

Should the css prop description fit the title a bit more? 

Maybe something like:

```suggestion
 * @prop --calcite-tab-content-space: Specifies the spacing of the component's content in the  `"default"` slot.
```

---

Shouldn’t this remain “padding”? Rest of the suggestion looks good.

---

Good catch 🎣 - updated the suggestion!